app/bundles/CampaignBundle/Executioner/Logger/EventLogger.php

```diff
@@ -43,7 +43,7 @@ class EventLogger
     /**
      * @var SummaryModel
      */
-    private $summaryModel;
+    public $summaryModel;
```

Comment:
This can stay private since there's a public getter.

Reply:
reverted back to private